common-api/src/test/java/org/pmiops/workbench/db/model/CommonStorageEnumsTest.java

```diff
@@ -51,4 +55,52 @@ public void testBijectiveStorageMapping() {
       assertThat(v).isEqualTo(fromStorage.apply(storageValue));
     }
   }
+
+  // domain ID is stringly-typed so special-case this
+
+  @Test
+  public void testDomainIdBijectiveStorageMapping() {
+    for (Domain v : Domain.values()) {
+      String storageValue = CommonStorageEnums.domainToDomainId(v);
+      assertWithMessage("unmapped enum value: " + v).that(storageValue).isNotNull();
+      assertThat(v).isEqualTo(CommonStorageEnums.domainIdToDomain(storageValue));
+    }
+  }
+
+  // copied from api/StorageEnumsTest because the above tests are not comprehensive
+  @Test
+  public void noMissingMapEntries() throws Exception {
+    for (Field f : CommonStorageEnums.class.getDeclaredFields()) {
+      if (f.getType() != BiMap.class) {
+        continue;
+      }
+
+      Class enumClass =
+          (Class) ((ParameterizedType) f.getAnnotatedType().getType()).getActualTypeArguments()[0];
+
+      Method enumToShort = null;
+      Method shortToEnum = null;
+      for (Method m : CommonStorageEnums.class.getDeclaredMethods()) {
+        if (m.getParameterTypes()[0].equals(enumClass)) {
+          enumToShort = m;
+        }
+
+        if (m.getReturnType().equals(enumClass)) {
+          shortToEnum = m;
+        }
+      }
+
+      // stringly typed map - test with testDomainIdBijectiveStorageMapping instead
+      if (enumToShort.getName().equals("domainIdToDomain")) {
+        continue;
+      }
+
+      for (Object e : enumClass.getEnumConstants()) {
```

Comment:
You're just testing BiMap at this point I think.

Reply:
This actually does catch bugs!  Its StorageEnumsTest counterpart caught one of mine today - that's what prompted me to do this.  Remove an entry from one of the BiMaps to observe this.

Reply:
No, I get what it's doing, and it's awesome that it catches bugs.

Is there a reason you have to copy this instead of making a generic utility function to use in both enum classes?

Reply:
We have a one-way dependency from /api -> /common-api so any common code would have to live in /common-api.  I don't want to add any new /common-api dependencies.  We'll be removing /common-api entirely, soon after Dover.

Reply:
But why did your new enum maps need to go in common at all?

Reply:
They did not.  I saw that the /api test saved my butt and was curious if the /common-api would have as well.  It would not have, so I fixed that here.